importScripts/refineryParameters/lowestLocation.ca_storage_locations.related.storageLocationSplitter.json

```diff
@@ -0,0 +1,100 @@
+{
+    "storageLocationType": "^lowestLocationType",
```

Comment:
Start by  assuming that the lowest location type is a box, so we don't need a parent for boxes.